build.gradle.kts

```diff
@@ -49,6 +49,7 @@ dependencies {
     testCompile("com.atlassian.performance.tools:infrastructure:[4.15.0,5.0.0)")
     testCompile("com.atlassian.performance.tools:virtual-users:[3.10.0,4.0.0)")
     testCompile("com.atlassian.performance.tools:jira-software-actions:[1.3.2,2.0.0)")
+    testCompile("com.atlassian.performance.tools:jira-actions:[3.11.0,4.0.0)")
```

Comment:
Nit
No need to bump it here. The new API is consumed only in `custom-vu`.
This adds an unnecessary constraint.

Reply:
I know it's not needed. I just wanted to have the same version in the whole project. Yore right there's no value in keeping it synced.